network/p2p/p2p_setup.go

```diff
@@ -47,6 +47,8 @@ const (
 	connectTimeout = time.Minute
 	// connectorQueueSize is the buffer size of the channel used by the connector
 	connectorQueueSize = 256
+	// inboundRatio is the ratio of inbound connections to outbound connections
+	inboundRatio = float64(0.8)
```

Comment:
It's probably `inboundShare` or `inboundFraction` and not `inboundRatio` (which would imply totalInboud = 0.8 * totalOutbound or something similar)